Exiled.API/Features/Pickups/Scp244Pickup.cs

```diff
@@ -0,0 +1,97 @@
+// -----------------------------------------------------------------------
+// <copyright file="Scp244Pickup.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features.Pickups
+{
+    using Exiled.API.Features.DamageHandlers;
+    using Exiled.API.Features.Items;
+
+    using InventorySystem.Items.Usables.Scp244;
+
+    using UnityEngine;
+
+    /// <summary>
+    /// A wrapper class for SCP-330 bags.
+    /// </summary>
+    public class Scp244Pickup : Pickup
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Scp244Pickup"/> class.
+        /// </summary>
+        /// <param name="itemBase">The base <see cref="Scp244DeployablePickup"/> class.</param>
+        public Scp244Pickup(Scp244DeployablePickup itemBase)
+            : base(itemBase)
+        {
+            Base = itemBase;
+        }
+
+        /// <summary>
+        /// Gets the <see cref="Scp244DeployablePickup"/> that this class is encapsulating.
+        /// </summary>
+        public new Scp244DeployablePickup Base { get; }
+
+        /// <summary>
+        /// Gets the speed of Scp244's too grow.
```

Comment:
`Gets the <see cref="Scp244"> GrowSpeed`

Reply:
i have put Scp244Pickup instead of Scp244 who send to Exiled.API.Feature.Items.Scp244 
or i need to put Scp244DeployablePickup ?

Reply:
yes, `Scp244Pickup`, i just wrote the abbreviation

Reply:
okay so it's done :)